### PR TITLE
explicitly pin thor to 1.1.0

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: debian/focal/foreman/rules [hunk 1]
```diff
@@ -22,6 +22,7 @@ build:
 	/bin/cp config/dynflow/worker.yml.example config/dynflow/worker.yml
 	/bin/mkdir -p ./vendor/cache
 	cd ./vendor/cache && /usr/bin/gem fetch thor -v '1.1.0'
```
Comment (ekohl) [inline debian/focal/foreman/rules:24]: Is this line still needed if we explicitly pin here?

Comment (evgeni) [reply debian/focal/foreman/rules:24]: probably not, but I wanted to start small ;)

**Review by ekohl — APPROVED** [review]

File: debian/buster/foreman/changelog [hunk 2]
```diff
@@ -21,7 +27,7 @@ foreman (3.1.0-3) stable; urgency=low
 
   * Do not restart Dynflow services on package upgrade
 
- -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 03 Nov 2021 11:26:17 -0400
+ -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 03 Nov 2021 11:26:21 -0400
```
Comment (evgeni) [inline debian/buster/foreman/changelog:30]: unrelated, but makes the changelogs identical across all branches ;)

**Comment by evgeni** [comment]
great, builds broken by https://github.com/theforeman/foreman/pull/8385/files#r781962143

**Comment by evgeni** [comment]
[test deb]

**Review by zjhuntin — APPROVED** [review]

**Comment by Toutzn** [comment]
I got this issue again in foreman 3.2.0 trying to migrate reports into new format

https://community.theforeman.org/t/migrating-reports-to-the-new-format-in-foreman-3-2/25846

foreman-rake host_reports:migrate
Traceback (most recent call last):
	12: from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
	11: from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
	10: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/setup.rb:10:in `<top (required)>'
	 9: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/ui/shell.rb:88:in `silence'
	 8: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/ui/shell.rb:136:in `with_level'
	 7: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/setup.rb:10:in `block in <top (required)>'
	 6: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
	 5: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
	 4: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
	 3: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
	 2: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
	 1: from /usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
/usr/share/rubygems-integration/all/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated thor 1.2.1, but your Gemfile requires thor 1.1.0. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)

What Is the solution to fix this issue?

BR. Torsten

**Comment by evgeni** [comment]
Hey @Toutzn,

what OS is this on? `/usr/sbin/foreman-rake` should be using the right bundler (`/usr/bin/bundle2.7`) in either case, but please double check.

Oh, and can you please open a dedicated topic for this on community.theforeman.org, discussing on a closed PR tends to be lost.

**Comment by Toutzn** [comment]
Hi, 

I opened a ticket under https://community.theforeman.org/t/you-have-already-activated-thor-1-2-1-but-your-gemfile-requires-thor-1-1-0/28005

hope this helps and I am right.

BR. Torsten

**Pull request merged**